app.py

```diff
@@ -126,16 +141,21 @@ def get_eventbrite_events(group_list):
     token = config.get('eventbrite', 'token')
     events = []
 
+    # Number of days to allow for past events
+    start_days = config.get('past_events', 'past_days')
+
     # the current date time in ISO8601 format
-    current_time = datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
+    current_time = (datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
```

Comment:
@ramonaspence  Is there an extra open parentheses on this line? I noticed it was changed and the only difference is an open parentheses.